deployment/Jenkinsfile

```diff
@@ -4,46 +4,33 @@ pipeline {
     stages {
         stage('Checkout') {
             steps {
-                checkout([$class: 'GitSCM', branches: [[name: '*/Dockerimages']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/tmarkesic/Auction-App.git']]])
+                checkout([$class: 'GitSCM', branches: [[name: '*/deployment']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/tmarkesic/Auction-App.git']]])
```

Comment:
Use `develop` as default branch. This is okay for testing, but in final, the main development branch needs to be used.